model/prelude.sail

```diff
@@ -178,22 +178,31 @@ val to_bits : forall 'l, 'l >= 0.(atom('l), int) -> bits('l)
 function to_bits (l, n) = get_slice_int(l, n, 0)
 
 infix 4 <_s
+infix 4 >_s
```

Comment:
This should probably be a separate commit?

Reply:
Yeah can do if you like - I think I just needed one missing operator for this and figured I'd add them all. I'll split it out.

Reply:
#406